bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -eux
+
+echo foobar | passwd --stdin root
+
+/usr/bin/systemctl stop firewalld NetworkManager kdump
+/usr/bin/systemctl disable firewalld NetworkManager kdump
+iptables -F
```

Comment:
This is supposed to disable the firewall during image installation? Why is that necessary?